jaxrs/src/main/java/org/jboss/aerogear/unifiedpush/rest/registry/installations/InstallationRegistrationEndpoint.java

```diff
@@ -75,6 +71,9 @@
     @Inject
     private GenericVariantService genericVariantService;
 
+    @Producer(topic = KAFKA_INSTALLATION_TOPIC)
+    private SimpleKafkaProducer<String, String> producer;
```

Comment:
Just a suggestion - it might be good to give the more descriptive name to the producer like installationMetricsProducer.